.infra/common.ts

```diff
@@ -31,6 +31,10 @@ export const workers: Worker[] = [
     topic: 'user-deleted',
     subscription: 'api.user-deleted-cio',
   },
+  {
+    topic: 'user-banned',
```

Comment:
We need to create this topic on the streams project. Also make sure to follow the name convention
```suggestion
    topic: 'api.v1.user-banned',
```

Reply:
I've got it locally in streams, will open PR for it later.

Was a bit unsure which naming scheme to go for, follow api.v1 like the rest, or just user-banned like the user-deleted etc 😬

Reply:
we have some legacy names :triggered: